src/main/java/jenkins/branch/OrganizationFolder.java

```diff
@@ -153,6 +159,13 @@
     private transient String facDigest;
 
     /**
+     * The {@link #buildStrategies} digest used to detect if we need to trigger a rescan on save.
+     *
+     * @since 2.0.12
+     */
+    private transient String bbsDigest;
```

Comment:
Might be better to have an equals/hashCode operation defined for the buildStrategies and use that for comparison, but this gets the job done.